javatests/arcs/android/storage/database/DatabaseImplTest.kt

```diff
@@ -435,6 +440,184 @@ class DatabaseImplTest {
     assertThat(entityOut).isEqualTo(entity)
   }
 
+  @Test
+  fun insertAndGet_entity_newEntityWithPrimitiveFields() = runBlockingTest {
```

Comment:
This test is not needed. (I deleted it and replaced it with the FixtureEntity test -- since you've added the duration field there you're already covered!)